pkg/endpointmanager/policymap_pressure.go

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package endpointmanager
+
+import (
+	"sync/atomic"
+	"time"
+
+	"github.com/cilium/cilium/pkg/endpoint"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+	"github.com/cilium/cilium/pkg/maps/policymap"
+	"github.com/cilium/cilium/pkg/metrics"
+	"github.com/cilium/cilium/pkg/trigger"
+)
+
+func (p *policyMapPressure) Update(ev endpoint.PolicyMapPressureEvent) {
+	val := ev.Value
+
+	log.WithField(logfields.Value, val).Debug("EndpointManager policymap received event")
+
+	cur := p.current.Load()
+	// TODO: Do we need to define a threshold for the comparison?
```

Comment:
Does this need to be addressed before merging?

Reply:
Dropped it. I think it's has minimal consequence anyway.